app/lib/features/profile/pages/my_profile_page.dart

```diff
@@ -87,26 +85,47 @@ class MyProfile extends ConsumerWidget {
     BuildContext context,
     WidgetRef ref,
   ) async {
-    final newUsername = await showDialog<String?>(
+    final TextEditingController newName = TextEditingController();
+    newName.text = profile.profile.displayName ?? '';
+
+    final newText = await showDialog<String>(
       context: context,
-      builder: (BuildContext context) => ChangeDisplayName(account: profile),
+      builder: (BuildContext context) => AlertDialog(
```

Comment:
It is unclear why this and the following changes have been done here... Why move an existing nice outside component into an inlined one? And what does that have to do with the rest of the PR?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/926/commits/7c5ae2ab2ca894a2e7d4db64362bf452ab42b21f